upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -404,6 +404,41 @@ if [[ $state_recorded == "0" && $(hostname) == "ncn-m001" ]]; then
     csi upgrade metadata --1-0-to-1-2 \
         --k8s-version ${KUBERNETES_VERSION} \
         --storage-version ${CEPH_VERSION}
+    
+    csi handoff bss-update-param --set psi=1
```

Comment:
do we need this here if we're also doing it below?

Reply:
No, cruft from initial commit.